### PR TITLE
[GAL-403] [GAL-404] Fix Safari bug

### PR DESCRIPTION
**Pull request by jakzaizzat**
#### Focusable on safari

| Before  | After  |
| ------------- | ------------- |
| CleanShot 2022-09-13 at 11 34 37 |  CleanShot 2022-09-13 at 11 35 11 |

#### Text overflow on feed safari

**Problem**
Seems like this bug only happens if there is more than one element under the `<Markdown/>` component. Example: Two paragraph element

**Solution**
Since, most of the child markdown element have their own spacing (padding/margin), we removed all the spacing and make all the markdown children to be inline instead of block (p, h1, h2, etc)

| Before  | After  |
| ------------- | ------------- |
| CleanShot 2022-09-13 at 11 37 37 | CleanShot 2022-09-13 at 11 36 36|

**Mobile**

**Safari 16**
IMG_2395C8212750-1

**Safari 15**
CleanShot 2022-09-20 at 16 42 57@2x

### CODE REVIEW

File: src/components/Feed/Events/CollectorsNoteAddedToTokenFeedEvent.tsx [hunk 1]
```diff
@@ -128,6 +128,11 @@ const StyledNote = styled(TitleM)`
   overflow: hidden;
   line-clamp: 4;
   display: -webkit-box;
+  text-overflow: ellipsis;
+
+  /* For safari: we need to specify the height */
+  /* Default font size * line-height * line-clamp */
+  height: calc(1rem * 1.5 * 4);
```
Comment (tbezman) [inline src/components/Feed/Events/CollectorsNoteAddedToTokenFeedEvent.tsx:135]: @jakzaizzat can you link to a stack overflow or something? why do we need to specify this height?

File: src/scenes/UserGalleryPage/UserGalleryCollections.tsx [hunk 2]
```diff
@@ -159,6 +159,9 @@ function UserGalleryCollections({ galleryRef, queryRef, mobileLayout }: Props) {
                   scrollTop={scrollTop}
                   deferredMeasurementCache={cache.current}
                   rowRenderer={rowRenderer}
+                  style={{
```
Comment (tbezman) [inline src/scenes/UserGalleryPage/UserGalleryCollections.tsx:162]: nice

File: cypress/integration/specs/home-spec.ts [hunk 3]
```diff
@@ -4,7 +4,6 @@ const home = new HomePage();
 
 describe('Homepage test', () => {
   beforeEach(() => {
-    cy.disconnectMetamaskWalletFromAllDapps();
```
Comment (tbezman) [inline cypress/integration/specs/home-spec.ts:7]: still failing :(

**Comment by tbezman** [comment]
also @jakzaizzat I'm seeing it like this on mobile safarai
Simulator Screen Shot - iPhone 14 Pro Max - 2022-09-12 at 20 55 45

**Comment by kaitoo1** [comment]
> also @jakzaizzat I'm seeing it like this on mobile safarai
> Simulator Screen Shot - iPhone 14 Pro Max - 2022-09-12 at 20 55 45

dat dynamic island 👀

**Comment by jakzaizzat** [comment]
@tbezman I assume its iOS 16 safari?

It looks fine on iOS 15 

CleanShot 2022-09-13 at 12 39 24

File: src/components/Feed/Events/CollectorsNoteAddedToTokenFeedEvent.tsx (hunk 1 above)
Comment (jakzaizzat) [reply src/components/Feed/Events/CollectorsNoteAddedToTokenFeedEvent.tsx:135]: @tbezman Seems like the `overflow hidden` didnt work very well in the new safari

https://stackoverflow.com/questions/70679732/css-line-clamp-does-not-work-in-safari-on-inner-block-level-elements/70683084#70683084

Found gist on line clamp safari

https://gist.github.com/EminQasimov/80ddcb7289d47a4d07caa1b4dc2bae83

**Comment by tbezman** [comment]
@jakzaizzat yeah :(

File: src/components/Feed/Events/CollectorsNoteAddedToTokenFeedEvent.tsx (hunk 1 above)
Comment (tbezman) [reply src/components/Feed/Events/CollectorsNoteAddedToTokenFeedEvent.tsx:135]: let's leave this in the code

**Comment by jakzaizzat** [comment]
@tbezman oh shitt. brb will update my phone to iOS 16 to test it 🏃🏻‍♀️

**Comment by Robinnnnn** [comment]
hey @jakzaizzat want to follow-up here, do you think a rebase with your latest changes would help?

File: src/components/Feed/Events/CollectorsNoteAddedToTokenFeedEvent.tsx (hunk 1 above)
Comment (jakzaizzat) [reply src/components/Feed/Events/CollectorsNoteAddedToTokenFeedEvent.tsx]: Updated! Found other solution without using this hacky height stuff

**Comment by jakzaizzat** [comment]
@tbezman @Robinnnnn  My bad. Forgot to add comment in this PR 🤦 

I've updated the changes and attached a screenshot too on mobile devices (Safari 15 & Safari 16)

**Review by Robinnnnn — APPROVED** [review]

**Pull request merged**